primitive/message.go

```diff
@@ -73,6 +74,13 @@ func NewMessage(topic string, body []byte) *Message {
 	}
 }
 
+func (msg *Message) SetDelayTimeLevel(level int) {
```

Comment:
add the description that the duration of each time level, could refer https://github.com/apache/rocketmq/blob/master/store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java#L129